server.js

```diff
@@ -21,7 +22,7 @@ mongoose.connect(process.env.MONGODB_URI || "mongodb://localhost/gamesdb");
 // Send every other request to the React app
 // Define any API routes before this runs
 app.get("*", (req, res) => {
-  res.sendFile(path.join(__dirname, "./client/build/index.html"));
+  res.sendFile(path.join(__dirname, "./client/public/index.html"));
```

Comment:
same for this with heroku

Reply:
I think we need the public - the build was put in when I ran the npm run build package